.github/workflows/git-artifacts.yml

```diff
@@ -289,12 +373,27 @@ jobs:
           *" ${{matrix.artifact.name}} "*|*" ${{matrix.artifact.name}}-${{matrix.arch.name}} "*) ;; # build this artifact
           *) echo "SKIP=true" >>$GITHUB_ENV;;
           esac
+      - name: Configure user
+        shell: bash
+        run:
+          USER_NAME="${{github.actor}}" &&
+          USER_EMAIL="${{github.actor}}@users.noreply.github.com" &&
+          mkdir -p "$HOME" &&
+          git config --global user.name "$USER_NAME" &&
+          git config --global user.email "$USER_EMAIL" &&
+          echo "PACKAGER=$USER_NAME <$USER_EMAIL>" >>$GITHUB_ENV
```

Comment:
We do not need `PACKAGER` here: that's only needed when building Pacman packages. Besides, I think we need to move this to the `build-arm64` job.